src/main/java/org/hawkular/client/android/backend/model/Alert.java

```diff
@@ -27,7 +29,8 @@
 import android.support.annotation.VisibleForTesting;
 
 public final class Alert implements Parcelable {
-    @SerializedName("alertId")
+    @RecordId
```

Comment:
I was using save() and that needed this. cause it needed a object with RecordId not null to place a PUT request.
Lemme check what is there for write() :)